src/policies.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+var axios = require('axios');
+var _ = require('lodash');
+var hcltojson = require('hcl-to-json');
+
+exports.listPolicies = function (req, res) {
+    let endpoint = `/v1/sys/policy`;
+    let vaultAddr = decodeURI(req.query['vaultaddr']);
+    let config = { headers: { 'X-Vault-Token': req.query['token'] } }
+
+    axios.get(`${vaultAddr}${endpoint}`, config)
+        .then((resp) => {
+            res.json(resp.data);
+        })
+        .catch((err) => {
+            console.error(err.stack);
+            res.status(err.response.status).send(err.response);
+        });
+}
+
+exports.getPolicy = function (req, res) {
+    let policyName = decodeURI(req.query['policy']);
+    let endpoint = `/v1/sys/policy/${policyName}`;
+    let vaultAddr = decodeURI(req.query['vaultaddr']);
+    let config = { headers: { 'X-Vault-Token': req.query['token'] } }
+
+    axios.get(`${vaultAddr}${endpoint}`, config)
+        .then((resp) => {
+            res.json(resp.data);
+        })
+        .catch((err) => {
+            console.error(err.stack);
+            res.status(err.response.status).send(err.response);
+        });
+}
+
+exports.updatePolicy = function (req, res) {
+    let policyName = decodeURI(req.query['policy']);
+    let endpoint = `/v1/sys/policy/${policyName}`;
+    let vaultAddr = decodeURI(req.query['vaultaddr']);
+    let config = { headers: { 'X-Vault-Token': req.query['token'] } }
+
+    //API requires an escaped JSON
+    let policy = _.get(req, "body.Policy");
+
+    // Attempt to parse into JSON incase a stringified JSON was sent
+    try {
+        policy = JSON.parse(policy)
+    } catch(e) { }
```

Comment:
its not meant to be an error, rather a check if the string is a valid json, and if it is, make it an object